=0.11.0.dev20170322155718

```diff
@@ -0,0 +1,6 @@
+Collecting coala
+  Downloading coala-0.10.0.tar.gz (2.0MB)
+    Complete output from command python setup.py egg_info:
+    coala supports only python 3.4 or later.
+    
```

Comment:
Hi, it got added when I ran tests I belive, not sure. I will reread the documentation and attempt to update the pull request so it passes the tests and making sure it doesn't add unneeded files.